src/components/ContextBreadcrumbs.tsx

```diff
@@ -93,30 +143,7 @@ export const ContextBreadcrumbs = ({
           <HomeLink color='gray' size={16} style={{ position: 'relative', left: -5, top: 2 }} />
         ) : null
       ) : (
-        <TransitionGroup>
-          {overflowArray.map(({ isOverflow, id, label }, i) => {
-            const subthoughts = ancestors(simplePath, id) as SimplePath
-            return (
-              <CSSTransition key={i} timeout={600} classNames='fade-600'>
-                {!isOverflow ? (
-                  <span>
-                    {i > 0 ? <span className='breadcrumb-divider'> • </span> : null}
-                    {subthoughts && <Link simplePath={subthoughts} label={label} />}
-                    {subthoughts && <Superscript simplePath={subthoughts} />}
-                  </span>
-                ) : (
-                  <span>
-                    <span className='breadcrumb-divider'> • </span>
-                    <span onClick={() => setEllipsize(false)} style={{ cursor: 'pointer' }}>
-                      {' '}
-                      ...{' '}
-                    </span>
-                  </span>
-                )}
-              </CSSTransition>
-            )
-          })}
-        </TransitionGroup>
+        renderBreadcrumbs()
```

Comment:
I'm not sure why you have wrapped the logic in a function `renderBreadcrumbs`, rather than using `<TransitionGroup childFactory={factoryManager}>{children}</TransitionGroup>` directly. 

If I'm missing something and the function is necessary, please use normal JSX syntax: `<TransitionBreadcrumbs>`. Otherwise, get rid of the wrapper function and define the `Breadcrumbs`, `children', and `factoryManager` in the `ContextBreadcrumbs` component itself. `renderBreadcrumbs` is using closure over `overflowArray` anyway, so it's not particularly modular.

Reply:
Sure !!